pyanaconda/ui/gui/spokes/datetime_spoke.py

```diff
@@ -564,6 +565,10 @@ def _initialize(self):
         # report that we are done
         self.initialize_done()
 
+    @property
+    def showable(self):
```

Comment:
I think that we should use `should_run` instead of `showable`. There is too much code that might crash before we check the `showable` property.